src/hooks/useHandleNFTTransfer.tsx

```diff
@@ -276,6 +289,24 @@ export function useHandleNFTTransfer() {
   const sourceTokenPublicKey = sourceParsedTokenAccount?.publicKey;
   const disabled = !isTargetComplete || isSending || isSendComplete;
   const handleTransferClick = useCallback(() => {
+    const telemetryProps: TelemetryTxEvent = {
+      fromChainId: sourceChain,
+      toChainId: targetChain,
+      fromTokenSymbol: sourceAsset,
+      toTokenSymbol: undefined,
+      fromTokenAddress: undefined,
+      toTokenAddress: undefined,
+      amount: undefined,
```

Comment:
```suggestion
  const targetChain = useSelector(selectAttestTargetChain);

....

    const handleAttestClick = useCallback(() => {
    const telemetryProps: TelemetryTxEvent = {
      fromChainId: sourceChain,
      toChainId: targetChain,
      fromTokenSymbol: undefined,
      toTokenSymbol: undefined,
      fromTokenAddress: sourceAsset,
      toTokenAddress: undefined,
      amount: undefined,
    };
    telemetry.on.transferInit(telemetryProps);
```

Reply:
not sure about this suggestion. looking closely it might be incorrect